sync/pacman/pacman-script

```diff
@@ -10,9 +10,13 @@ fi
 
 REPO_DIR="${1}"
 YAMLFILE="${1}/pacman.yaml"
-TEMPORARY_DIR=$(mktemp -d)/
+TEMPORARY_DIR="${REPO_DIR}/.tmp"
```

Comment:
Does this mean the temp dir will be in the repo if i run this locally?

Then i guess the `# .tmp was moved so that the files in the repo dir were cleaner.` isn't valid anymore.

Does this mean i manually have to remove the temp dir after each run?

Reply:
If this is an issue with finding the tmp dir afterwards simply outputting in by echo might be enough?

Reply:
It can be added to the gitignore, but it will show up in IDEs.

Reply:
Yes, the temp dir will be in the repo and is currently in the .gitignore file.

> Then i guess the `# .tmp was moved so that the files in the repo dir were cleaner.` isn't valid anymore?

It's partially valid and partially invalid! The temp dir was in the repo, then got moved out to a random folder in `/tmp`, then got moved back in to the repo again. I can update the comment to be more correct.

> Does this mean i manually have to remove the temp dir after each run?

No, the script cleans the temp dir before each run. It could also clean it up afterwards as well if we wanted.

> If this is an issue with finding the tmp dir afterwards simply outputting in by echo might be enough?

For `mv` to be faster than `cp` it needs to be on the same filesystem (and mounted with the same docker volume command), that is why it is in the repository rather than in `/tmp`.

Reply:
Cool, thanks for clarifying. Marking this as resolved.